bin/pymultimonaprs

```diff
@@ -91,8 +91,15 @@ def bc():
 		sleep(config['beacon']['send_every'])
 
 logger.info("Starting pymultimonaprs")
+				
+if 'prefered_protocol' in config:
+	prefered_protocol = config['prefered_protocol'].lower()
```

Comment:
Thanks, i'm not at all a Python coder so i didn't know about this one. Changed it to your solution.